pandas_datareader/tests/yahoo/test_yahoo.py

```diff
@@ -133,7 +133,7 @@ def test_get_data_multiple_symbols(self):
 
     @pytest.mark.parametrize('adj_pr', [True, False])
     def test_get_data_null_as_missing_data(self, adj_pr):
-        #workaround as skip_on_exception decorator changes signature
```

Comment:
I think @bashtage gave you everything there, so there is nothing else to do but copy and paste.  What happens when you do this?